pkg/env/max_parallel_scans.go

```diff
@@ -4,4 +4,8 @@ var (
 	// MaxParallelImageScanInternal sets the max number of parallel scans collectively on the ScanImageInternal, EnrichLocalImageInternal,
 	// and GetImageVulnerabilitiesInternal endpoints and separately sets the max active local scans in a secured cluster
 	MaxParallelImageScanInternal = RegisterIntegerSetting("ROX_MAX_PARALLEL_IMAGE_SCAN_INTERNAL", 30)
+
+	// MaxParallelDelegatedScanInternal defines the maximum number of parallel delegated scans initiated from Central.
+	// It must be set lower than the collective limit for ScanImageInternal, EnrichLocalImageInternal, and GetImageVulnerabilitiesInternal endpoints.
```

Comment:
how come? What is this max value?

Reply:
I am not sure about proper name. It should be requests from ad-hoc roxctl.

Reply:
5 seems to be a bit maximum value a bit arbitrary but I decided after discussion with @dcaravel

Reply:
It was arbitrary - thought process was lets not crash scanner-slim / scanner-v4-indexer by ensuring we do not send more than the current max with this change. Would be better to have some data to back these #'s up, so right now relying on 'what is working in user environments' until we have test metrics avail.